### PR TITLE
Adding ImageStream to do oc tag and oc label commands to the jenkins …

### PR DESCRIPTION
**Pull request by gsampaio-rh**
Resolve https://github.com/rht-labs/enablement-docs/issues/236

@oybed @springdo @tylerauerbeck @eformat

### CODE REVIEW

File: templates/jenkins-s2i.yml [hunk 1]
```diff
@@ -40,6 +40,22 @@ objects:
...
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    labels:
+      role: jenkins-slave
+    name: jenkins-slave-npm
+    namespace: ${IMAGE_STREAM_NAMESPACE}
+  spec:
+    tags:
+    - annotations: null
```
Comment (springdo) [inline templates/jenkins-s2i.yml]: think this can go?

Comment (oybed) [reply templates/jenkins-s2i.yml]: Done

**Comment by oybed** [comment]
Also please add the PR number for the https://github.com/rht-labs/enablement-docs changes

**Comment by gsampaio-rh** [comment]
https://github.com/rht-labs/enablement-docs/pull/369 Docs PR

**Pull request merged**